README.md

```diff
@@ -113,12 +113,12 @@ Have a bug or a feature request? Please first read the [issues guidelines](https
 
 ## Documentation
 
-Official plotly.js documentation is hosted on [plot.ly/javascript](https://plot.ly/javascript).
+Official plotly.js documentation is hosted on [plotly.com/javascript](https://plotly.com/javascript).
 
-These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](http://jekyllrb.com) and publicly hosted on GitHub Pages.
+These pages are generated by the Plotly [documentation repo](https://github.com/plotly/documentation/tree/gh-pages) built with [Jekyll](https://jekyllrb.com/) and publicly hosted on GitHub Pages.
```

Comment:
can you update these to replace `documentation` with `graphing-library-docs` please? The new repo is at https://github.com/plotly/graphing-library-docs